src/js/media-controller.js

```diff
@@ -444,6 +444,9 @@ const setAttr = (child, attrName, attrValue) => {
     if (attrValue) return child.setAttribute(attrName, '');
     return child.removeAttribute(attrName);
   }
+  if (typeof attrValue === 'number' && Number.isNaN(attrValue)) {
+    return child.removeAttribute(attrName);
+  }
```

Comment:
Makes sense 👍🏼

Reply:
If you use `Number.isNaN` the type check is supposed to be not necessary.

Reply:
Yup good callout. Was modeling it on "type" + specific behavior, but it's redundant in this case. Will remove before merging.